tests/rxtx/tests.c

```diff
@@ -181,7 +182,44 @@ static void test_rxtx_rxtx_to_rxtx(void)
 	}
 	pg_graph_destroy(g);
 }
+static void mytx_empty(struct pg_brick *brick,
+                 struct pg_rxtx_packet *tx_burst,
+                 uint16_t *tx_burst_len,
+                 void *private_data)
+{
+	struct pg_rxtx_packet empty[PG_MAX_PKTS_BURST];
+	empty->data = 0;
+	empty->len = 0;
+	*tx_burst = *empty;
+	*tx_burst_len = 0;
+}
+
+static void test_rxtx_empty_burst(void)
+{
+	struct pg_error *error = NULL;
+	struct pg_brick *rxtx;
+	struct pg_brick *fail;
+	struct test_rxtx pd;
+	struct pg_graph *g;
+
+	rxtx = pg_rxtx_new("rxtx", NULL, &mytx_empty, (void*) &pd);
+	fail = pg_fail_new("fail", &error);
+	CHECK_ERROR(error);
+
+	pg_brick_link(rxtx, fail, &error);
+	CHECK_ERROR(error);
+
+	g = pg_graph_new("graph", rxtx, &error);
+        CHECK_ERROR(error);
+        g_assert(g);
 
+	pg_graph_poll(g, &error);
+	CHECK_ERROR(error);
+
+	pg_brick_destroy(rxtx);
```

Comment:
the point to have a graph it that it will free bricks for you... in think, so pg_brick_destroy should be useless